modules/imaging_browser/php/mrinavigation.class.inc

```diff
@@ -46,11 +46,8 @@ class MRINavigation
     {
         $this->_sessionID           = $sessionID;
         $this->_FilteredSessionList = $_SESSION['State']
-            ->getProperty('mriSessionsListed');
+            ->getProperty('mriSessionsListed') ?? [];
 
-        if (!is_array($this->_FilteredSessionList)) {
```

Comment:
I removed this with the assumption that $_SESSION['State']->getProperty('mriSessionsListed') is either of type array or null.

Reply:
Update -- I will adjust to take your previous comment into account.